scripts/movement/build-push-image

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
+set -e
+
 FILE_PATH=./docker/build/$1/Dockerfile
-./scripts/docker/build-push-image $FILE_PATH mvlbs $1
\ No newline at end of file
+
+./scripts/docker/build-push-image $FILE_PATH mvlbs $1
```

Comment:
I'm afraid we don't have a `movement/Dockerfile` in master any more. Should we remove this workflow or let it be disabled to put something else later instead?

Is there currently a value in building containers for every git push?